docs/time/index.rst

```diff
@@ -415,33 +446,45 @@ that work on `~numpy.ndarray` instances can be used.  E.g., one can reshape
 
 Note that similarly to the `~numpy.ndarray` methods, all but
 :meth:`~astropy.time.Time.flatten` try to use new views of the data,
-with the data copied only if that it is impossible (as discussed, e.g., in
-the documentation for numpy :func:`~numpy.reshape`).
+with the data copied only if that is impossible (as discussed, for example, in
+the documentation for ``numpy`` :func:`~numpy.reshape`).
+
+.. EXAMPLE END
 
 Some arithmetic methods are supported as well: :meth:`~astropy.time.Time.min`,
 :meth:`~astropy.time.Time.max`, :meth:`~astropy.time.Time.ptp`,
 :meth:`~astropy.time.Time.sort`, :meth:`~astropy.time.Time.argmin`,
 :meth:`~astropy.time.Time.argmax`, and :meth:`~astropy.time.Time.argsort`.
-E.g.::
+
+.. EXAMPLE START: Reshaping Time Instances Using Arithmetic Methods
```

Comment:
This is not really reshaping. I would write "Applying Arithmetic Methods to Time Instances" (and similarly in the actual sentence seen by the reader below)